pre-build

```diff
@@ -24,6 +24,12 @@ if [ -f $DIR/apt-packages ]; then
     apt-get install -y \$PACKAGES
     echo "-----> Injected packages: \$PACKAGES"
 fi
+if [ -d $DIR/dpkg-packages ]; then
+    for pkg in $DIR/dpkg-packages/*.deb; do
+        dpkg -i $DIR/dpkg-packages/\$pkg;
```

Comment:
It was working when you used `ls` to list the deb files, but now the glob `$DIR/dpkg-packages/*.deb` includes the path.